src/shared/bridge.js

```diff
@@ -65,10 +66,7 @@ export default class Bridge {
     channel.call('connect', token, ready);
 
     // Store the newly created channel in our collection
-    this.links.push({
-      channel,
-      window: source,
```

Comment:
Was `window` not used?

Reply:
It was not used.

I am happy to make use of it, if you wish.

Reply:
If we're not using it then it's fine to remove it.